server/index.js

```diff
@@ -70,11 +142,13 @@ app.get('/api/users/:userId', (req, res, next) => {
 app.put('/api/users/:userId', uploadsMiddleware, (req, res, next) => {
   const { userId } = req.params;
   const { name, position, location, availability } = req.body;
-  const profilePicUrl = `/images/${req.file.filename}`;
+  const profilePicUrl = req.file
+    ? `/images/${req.file.filename}`
+    : null;
```

Comment:
Instead of `/api/users/:userId`, change this to `/api/me` and register an authorization middleware before it. We don't want people to randomly be able to edit each other's profiles by sending HTTP requests to your server.

See `api-authorization`.